### ISSUE TITLE
Broken in 10.3.2

### ISSUE THREAD

**Issue by higgsy** [Geta/404handler#24]
Hi,

I have found that a runtime error is thrown when using it on version 10.3.2 (I can't confirm whether it's broken on versions between 10.1.0 and 10.3.2). I thought it might just be the project we're working on so I tried it on AlloyTech too and it still throws the error. The issue in the browser is:

Method not found: 'Void EPiServer.Web.InitializationModule.add_FirstBeginRequest(System.EventHandler)'.

I looked at the logs on our project and entire stack trace is:

2017-01-30 13:42:14,715 [73] ERROR EPiServer.Framework.Initialization.InitializationEngine: Initialize action failed for 'Initialize on class BVNetwork.FileNotFound.Redirects.CustomRedirectInitialization, BVNetwork.EPi404, Version=3.1.1.27532, Culture=neutral, PublicKeyToken=null'
System.MissingMethodException: Method not found: 'Void EPiServer.Web.InitializationModule.add_FirstBeginRequest(System.EventHandler)'.
   at BVNetwork.FileNotFound.Redirects.CustomRedirectInitialization.Initialize(InitializationEngine context)
   at EPiServer.Framework.Initialization.Internal.ModuleNode.Execute(Action a, String key)
   at EPiServer.Framework.Initialization.InitializationEngine.InitializeModules()
System.MissingMethodException: Method not found: 'Void EPiServer.Web.InitializationModule.add_FirstBeginRequest(System.EventHandler)'.
   at BVNetwork.FileNotFound.Redirects.CustomRedirectInitialization.Initialize(InitializationEngine context)
   at EPiServer.Framework.Initialization.Internal.ModuleNode.Execute(Action a, String key)
   at EPiServer.Framework.Initialization.InitializationEngine.InitializeModules()

I cloned the codebase, however it's strange because I can't even find a BVNetwork.FileNotFound.Redirects.CustomRedirectInitialization namespace/class.

Hopefully you can diagnose the issue.

Thanks for the great plugin.

Regards,
Al

**Comment by M15sy** [comment]
Hi, 
This is an issue for us too and is preventing us from updating to the latest EPiServer update.
It does not look like a trivial fix but are there plans to make the plugin compatible?
Thanks

**Comment by evest** [comment]
Hi, sorry for the late reply, but did you resolve this? The 404 module version from that error message seems very old. Have you tried upgrading?

Btw. also check your modulesbin folder for any old assembly left-overs if you're working on an upgraded site.

**Comment by madsstorm** [comment]
Hi,

I saw a similar issue, when starting a new development project on an old developer PC.
The new project uses BVNetwork.EPi404.dll version 10.1.1.0, but the error message mentioned version 3.1.1.14788.
I had previously used version 3.1.1.14788 of BVNetwork.EPi404.dll in another project on the same developer PC.
I found the old version in "C:\ProgramData\Red Gate\.NET Reflector\DevPath" and deleted it, which made my new project work.

I can see the "AssemblyInfo.Cms10.cs" contains an "AssemblyVersion" but no "AssemblyFileVersion". I am not sure if this is the cause, but it looks unusual.
